commit d684b8082f598a02fac8b8b25ec8d6fa80d7f14e
Author: Friedrich W. H. Kossebau <kossebau@kde.org>
Date:   Sat Feb 21 03:55:17 2015 +0100

    Fix installation path for Kasten includes (should be without number postfix)
    
    fixup for 2b6e05cc0a84db1e42d9bc84948afd5c557c4ae7

diff --git a/kasten/CMakeLists.txt b/kasten/CMakeLists.txt
index 4ace88d..140f925 100644
--- a/kasten/CMakeLists.txt
+++ b/kasten/CMakeLists.txt
@@ -13,7 +13,7 @@ set( OKTETAKASTEN_MINOR_VERSION 3 )
 set( OKTETAKASTEN_PATCHLEVEL_VERSION 0 )
 set( OKTETAKASTEN_VERSION "${OKTETAKASTEN_MAJOR_VERSION}.${OKTETAKASTEN_MINOR_VERSION}.${OKTETAKASTEN_PATCHLEVEL_VERSION}" )
 set( OKTETAKASTEN_ABI_VERSION 1 )
-set( OKTETAKASTEN_INCLUDE_INSTALL_DIR  ${INCLUDE_INSTALL_DIR}/kasten${KASTEN_ABI_VERSION}/okteta${OKTETAKASTEN_ABI_VERSION} )
+set( OKTETAKASTEN_INCLUDE_INSTALL_DIR  ${INCLUDE_INSTALL_DIR}/kasten/okteta ) #${KASTEN_ABI_VERSION}/okteta${OKTETAKASTEN_ABI_VERSION} )
 
 set( oktetakastencore_LIB         oktetakastencore )
 set( oktetakastengui_LIB          oktetakastengui )
diff --git a/libs/kasten/CMakeLists.txt b/libs/kasten/CMakeLists.txt
index 29af950..94decdf 100644
--- a/libs/kasten/CMakeLists.txt
+++ b/libs/kasten/CMakeLists.txt
@@ -5,7 +5,7 @@ set( KASTEN_MINOR_VERSION 3 )
 set( KASTEN_PATCHLEVEL_VERSION 0 )
 set( KASTEN_VERSION "${KASTEN_MAJOR_VERSION}.${KASTEN_MINOR_VERSION}.${KASTEN_PATCHLEVEL_VERSION}" )
 set( KASTEN_ABI_VERSION 3 )
-set( KASTEN_INCLUDE_INSTALL_DIR  ${INCLUDE_INSTALL_DIR}/kasten${KASTEN_ABI_VERSION} )
+set( KASTEN_INCLUDE_INSTALL_DIR  ${INCLUDE_INSTALL_DIR}/kasten ) #${KASTEN_ABI_VERSION} )
 
 set( kastencore_LIB         kastencore )
 set( kastengui_LIB          kastengui )
diff --git a/libs/kasten/includes/CMakeLists.txt b/libs/kasten/includes/CMakeLists.txt
index 29066a3..012e026 100644
--- a/libs/kasten/includes/CMakeLists.txt
+++ b/libs/kasten/includes/CMakeLists.txt
@@ -126,6 +126,6 @@ set( kasten_CamelCase_HDRS
 )
 
 install( FILES ${kasten_CamelCase_HDRS}
-  DESTINATION ${INCLUDE_INSTALL_DIR}/Kasten${KASTEN_ABI_VERSION}
+  DESTINATION ${INCLUDE_INSTALL_DIR}/Kasten #${KASTEN_ABI_VERSION}
   COMPONENT   Devel
 )
